src/rewards/veterans.rs

```diff
@@ -117,14 +114,26 @@ pub fn calculate_veteran_advisors_incentives(
         &rankings_per_vca,
         eligible_rankings_per_vca.clone(),
         reputation_thresholds,
-        |agreement| disagreement_modifier(agreement, THRESHOLDS, REPUTATION_DISAGREEMENT_MODIFIERS),
+        |agreement| {
+            disagreement_modifier(
+                agreement,
+                agreement_rate_thresholds.clone(),
+                REPUTATION_DISAGREEMENT_MODIFIERS,
```

Comment:
This is still hardcoded and does not work if we have more (or less) than 3 thresholds